engine/runtime/src/main/java/org/enso/interpreter/node/callable/resolver/HostMethodCallNode.java

```diff
@@ -224,6 +226,20 @@ Object resolveHostMethod(
               .error()
               .makeUnsupportedArgumentsError(e.getSuppliedValues(), e.getMessage()),
           this);
+    } catch (AbstractTruffleException ex) {
+      CompilerDirectives.transferToInterpreter();
+      if (ex.getMessage().contains("Unsupported operation identifier")) {
```

Comment:
This code is trying to recognize `HostException` (a non-public Graal class) and extract enough information from it to re-create a `PanicException`.

This is pretty ugly and more over wrong...